app/models/log.rb

```diff
@@ -106,7 +106,12 @@ def blank_invalid_non_setup_fields!
       errors.each do |error|
         next if setup_ids.include?(error.attribute.to_s)
 
-        public_send("#{error.attribute}=", nil)
+        question = form.questions.find { |q| q.id == error.attribute.to_s }
+        if question&.type == "checkbox"
+          question.answer_keys_without_dividers.each { |attribute| public_send("#{attribute}=", nil) }
+        else
```

Comment:
How come we're clearing the values here? Would this impact anything beyond bulk upload?

Reply:
Nope, this method is only called by the validators/log_creators for bulk upload, we were already clearing it on line 109 this just makes sure to actually clear ALL the values for multiple checkbox questions (welcome back!)